app/lib/encryption/encryption.js

```diff
@@ -240,15 +240,15 @@ class Encryption {
 						msg,
 						tmsg
 					});
-					if (message._hasPendingUpdate) {
-						console.log(message);
-						return;
+					try {
```

Comment:
Instead of relying on `_hasPendingUpdate` to prevent the error, let's wrap the whole call on `try/catch` and handle the error.